src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -347,15 +351,22 @@ const StyledPageContent = styled(PageContent)`
 
 const ChartPanel = styled(Panel)`
   padding: ${space(2)};
+  margin-bottom: 0;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
 `;
 
 const ChartHeader = styled('header')`
   margin-bottom: ${space(1)};
-  display: grid;
-  grid-template-columns: 1fr max-content;
 `;
 
-const ChartActions = styled('div')``;
+const ChartActions = styled(Panel)`
```

Comment:
Can we actually put the `ChartActions` inside the ChartPanel, instead of killing the border radius

Reply:
I think it might be marginally simpler.

Reply:
thanks, used PanelFooter and PanelBody